integrations/database/postgresql/migrations/03_uuid_idx_log_url.down.sql

```diff
@@ -0,0 +1,2 @@
+DROP INDEX log_url_uuid_idx;
+CREATE INDEX log_url_uuid_idx ON resource USING btree (uuid);
```

Comment:
what about 
```suggestion
CREATE INDEX IF NOT EXISTS log_url_uuid_idx ON resource USING btree (uuid);
```

That way this allows us or other to have this index in place before the migration and don't make it crash.

We can do the same with the rest:

```
DROP INDEX IF EXISTS log_url_uuid_idx;
```

Reply:
done